### PR TITLE
Docs reorganization

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
re-organize docs - each higher section now indicates to a page that contains a table of content, which help users find their area of interest easier

### CODE REVIEW

File: docs/wallet/overview.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+id: overview
+title: Wallets - Overview
+---
+
+## Elrond Wallets
+
+The private key associated to an address (erd1...) needs to be store in a specific format. In order to use that private
```
Comment (radumojic) [inline docs/wallet/overview.md:8]: needs to be store**d**

File: docs/developers/overview.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+## Table of contents
+
+### Tutorials
+
+Below is a list of tutorials that one can you to get a full experience of building on Elrond
+
+| Name                                                                                | Description                                                      |
+|-------------------------------------------------------------------------------------|------------------------------------------------------------------|
+| [Build your first dApp in 15 minutes](/developers/tutorials/your-first-dapp)        | Video + written tutorial on how to create your first dApp.       |
+| [Build a microservice for your dApp](/developers/tutorials/your-first-microservice) | Video + written tutorial on how to create for your microservice. |
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/developers/overview.md:24]: fixed

File: docs/validators/overview.md [hunk 3]
```diff
@@ -1,15 +1,65 @@
...
+|-----------------------------------------------------------------|-------------------------------------|
+| [Validator keys](/validators/key-management/validator-keys)     | Learn about a validator key.        |
+| [Wallet keys](/validators/key-management/wallet-keys)           | Learn about a wallet key.           |
+| [Protecting your keys](/validators/key-management/protect-keys) | Learn how you can secure your keys. |
+
+### Staking
+
+| Name                                                                                | Description                                                   |
+|-------------------------------------------------------------------------------------|---------------------------------------------------------------|
+| [Staking, unstaking and unjailing](/validators/staking/staking-unstaking-unjailing) | Learn about how to stake, to unstake or unjail a Node.        |
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/validators/overview.md:28]: fixed

File: docs/wallet/overview.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+The private key associated to an address (erd1...) needs to be store in a specific format. In order to use that private 
+key to sign transactions and perform various actions, one needs a wallet.
+
+There are multiple ways you can store your funds. This page will present some of them.
+
+## Table of contents
+
+| Name                                                    | Description                                                                                                         |
+|---------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
+| [Maiar App](https://maiar.com/)                         | Digital wallet and global payments app that allows you to exchange and securely store money on your mobile phone. . |
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/wallet/overview.md:17]: fixed

**Review by radumojic — APPROVED** [review]

**Pull request merged**